docs/upgrades.md

```diff
@@ -117,6 +117,54 @@ delta, but only:
 when performing the upgrade.
 {% endhint %}
 
+### vX.X.X to v0.32.0
```

Comment:
Nit: what does X.X.X mean? Does it mean any versions? If so, shall we make it clear as such? eg "vX.X.X (any version) to v0.32.0". At the first glimpse, I wasn't sure.

Reply:
Yes, it means "any version to 0.32.0". This formulation already exists in this doc, so I just reused the same format.